bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -4523,28 +4523,22 @@
     		INSERT INTO role_privilege (role, privilege) VALUES ('Anonymous', 'Get Locations');
     	</sql>
     </changeSet>
-    <changeSet id="bahmni-core-202011150948" author="Buvaneswari">
+    <changeSet id="bahmni-core-20201210148" author="Buvaneswari">
         <preConditions onFail="MARK_RAN">
             <not><tableExists tableName="form_privilege"/></not>
         </preConditions>
         <comment>Introducing form_privilege table as part of BAH_790 Privilege based access to forms</comment>
         <sql>
-            CREATE TABLE form_privilege (
+            CREATE TABLE form_privilege(
```

Comment:
Why aren't we using typical liquibase <createTable> syntax?

<createTable tableName="form_privilege">
            <column name="form_privilege_id" autoIncrement="true" type="INT">
                <constraints primaryKey="true"/>
            </column>
       ....
</createTable>